networking/multiple_networks/primary_networks/creating-primary-nad.adoc

```diff
@@ -0,0 +1,36 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="creating_primary_networks_using_NAD"]
+= Creating primary networks using NetworkAttachmentDefinitions
+include::_attributes/common-attributes.adoc[]
+:context: understanding-multiple-networks
+
+toc::[]
+
+The following sections explain how to create and manage additional primary networks using the `NetworkAttachmentDefinition` (NAD) resource.
```

Comment:
It seems to me this "primary_networks" talking about ovn default network.   But we have 'primary network' in udn concept, this is a bit confusing.  cc @tssurya  Can you help check? Thanks!